content/openapi/introduction.md

```diff
@@ -132,6 +133,39 @@ const document = SwaggerModule.createDocument(app, options, {
 });
 ```
 
+### Setup options
+
+While calling setup method, it is possible to have some extra options. These options should be of type `SwaggerCustomOptions`, which can be the following:
+
+```TypeScript
+export interface SwaggerCustomOptions {
+    explorer?: boolean;
+    swaggerOptions?: any;
```

Comment:
Would this be better typed as `Record<string, any>`? `any` itself is a bit vague and makes it seem a `boolean` or a `number` fits here, when it should really be an object.

Reply:
This is the exact thing that I copied from the source code. Despite `SwaggerDocumentOptions` that has some comments and more details, `SwaggerCustomOptions ` has nothing. Btw, if you want I can add `Record<string, any>` instead of any. Maybe I can add it to `SwaggerCustomOptions` in another pull request.

Reply:
@vahidvdn I like this idea! feel free to create a PR to swagger repo

Reply:
@kamilmysliwiec actually the best idea is to define a type for `swaggerOptions` parameter. For example, the developers have no idea to use `persistAuthorization` without a type definition. But I really have no idea what all parameters are available for `swaggerOptions`.
Btw, for this PR if still needs more changes please let me know. Thanks